bot/src/main/java/com/almightyalpaca/discord/jdabutler/Bot.java

```diff
@@ -99,6 +99,7 @@ public static void main(final String[] args) throws JsonIOException, JsonSyntaxE
 
         final String token = Bot.config.getString("discord.token", "Your token");
         builder.setToken(token);
+        builder.setChunkingFilter(ChunkingFilter.NONE);
```

Comment:
could probably also disable some caches (voice at least)

Reply:
This can be done in another PR. I'm not completely sure what is used and I don't want to make any breaking changes.

Reply:
Sure!